CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+
+- Linear Progress component
+- Circular Progress component
+- Circular Gradient component
```

Comment:
We usually do no mention the "helper" components like gradient component in the changelog.